generate-docs/tools/WhatsNew.ts

```diff
@@ -85,44 +85,34 @@ class APISet {
         return found;
     }
 
-    public containsField(field: FieldStruct): boolean {
+    public containsField(clas: ClassStruct, field: FieldStruct): boolean {
```

Comment:
"class" instead "clas"?

Reply:
`class` is a reserved keyword. I'm open to suggestions, though.

Reply:
Right, forgot that. "className" was my usual goto.